core/node.py

```diff
@@ -133,3 +154,27 @@ def get_node_signature(validator_id):
         return payload['signature']
     else:
         return payload
+
+
+def backup(path):
+    backup_filepath = get_backup_filepath(path)
+    create_backup_archive(backup_filepath)
+
+
+def get_backup_filename():
+    time = datetime.datetime.utcnow().strftime("%Y-%m-%d-%H:%M:%S")
+    return f'{BACKUP_ARCHIVE_NAME}-{time}.tar.gz'
+
+
+def get_backup_filepath(base_path):
+    return os.path.join(base_path, get_backup_filename())
+
+
+def create_backup_archive(backup_filepath):
+    cmd = shlex.split(f'tar -zcvf {backup_filepath} -C {HOME_DIR} .skale')
```

Comment:
May be it's better to use tarfile module ?

Reply:
there were some problems in skale-admin with tarfile module (in `logs dump` cmd), so I decided to use system `tar` package here too

Reply:
ok, let's leave as is for now